http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1694,47 +1742,36 @@ components:
     ListRuntimeResponse:
       description: ""
       required:
+        - id
         - runtimeName
         - googleProject
         - serviceAccount
-        - machineConfig
+        - runtimeConfig
         - status
         - labels
-        - autopauseThreshold
-        - defaultClientId
       properties:
-        internalId:
-          type: string
-          description: Internal resource ID of the runtime
+        id:
```

Comment:
we have both `id` and `internalId`, just want to double check that this is returning `id` but not `internalId`?

Reply:
Yeah, if you trace it, it ends at this encoder https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala#L379

verified here:
![Screen Shot 2020-04-23 at 5 55 12 PM](https://user-images.githubusercontent.com/6465084/80153488-9f6eab80-858b-11ea-9d8a-dc9487841676.png)